spec/Section 3 -- Type System.md

```diff
@@ -920,7 +921,8 @@ of rules must be adhered to by every Object type in a GraphQL schema.
          returns {true}.
       4. If argument type is Non-Null and a default value is not defined:
          1. The `@deprecated` directive must not be applied to this argument.
-3. An object type may declare that it implements one or more unique interfaces.
+3. An object type may declare that it implements a set of one or more unique
```

Comment:
Wouldn't this also be an ordered set of interfaces?

Reply:
Yep; great catch - 602d319